src/VolleyManagement.Backend/VolleyManagement.Services/PlayerService.cs

```diff
@@ -36,21 +37,27 @@ public class PlayerService : IPlayerService
         /// <param name="playerRepository"> The player repository </param>
         /// <param name="getTeamByIdQuery"> Get By ID query for Teams</param>
         /// <param name="getPlayerByIdQuery"> Get By ID query for Players</param>
+        /// <param name="getPlayerTeamQuery"></param>
+        /// <param name="getFreePlayers"></param>
         /// <param name="getAllPlayersQuery"> Get All players query</param>
         /// <param name="getTeamByCaptainQuery">Get Player by Captain query</param>
         /// <param name="authService">Authorization service</param>
         public PlayerService(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than the 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS107)

Reply:
Ignore this one